CONTRIBUTING.md

```diff
@@ -131,6 +135,8 @@ The first 3 steps are the same, but this time, instead of running the tests in h
 yarn 
 # Build Elements itself
 yarn build
+# Copy the contents of predefined example application and make it use the local build
```

Comment:
Is this maybe something that build should do?

Reply:
It would definitely decrease the number of commands but increase the build time. I don't think it's necessary to copy the whole integration folder each time we run a build

Reply:
Did you consider manipulating original package instead of using that integration folder? Something like modifying package.json temporarily and after example build process reverting it? Or having already prepared `package.json_integration` and replace original one with it before build and reverting after?

Reply:
Yes, I did. The downside is that beacause Gatsby doesn't allow custom output folder for builds, you still would need to copy some stuff around, if you for instance would like to run both of these integrations ('public' and 'development') at the same time.

I like the idea of providing the pre-made `package.json` though 👍

Reply:
But can't the old build be replaced with new one? Would it really require using custom output folders for that?

Reply:
It can, but are there any advantages to that solution? `examples` folder might become a bit confusing for users with that approach IMHO. Each time they would want to compare their work to an already working example (which is currently in `examples`), they would need to build both of them.

Plus, if we create a separate folder for developed integrations, we get much more than just ability to run e2e, without worrying that it can affect existing ('public') examples.